frontend/src/app/pages/exercises/exercise/shared/emergency-operations-center/send-alarm-group-interface/send-alarm-group-interface.component.ts

```diff
@@ -67,15 +69,40 @@ export class SendAlarmGroupInterfaceComponent implements OnDestroy {
         targetTransferPointId = value;
     }
 
+    public get firstVehiclesToOtherLocationTransferPointId() {
+        return firstVehiclesToOtherLocationTransferPointId;
+    }
+    public set firstVehiclesToOtherLocationTransferPointId(
+        value: UUID | undefined
+    ) {
+        firstVehiclesToOtherLocationTransferPointId = value;
+    }
+
+    public get firstVehiclesToOtherLocationCount() {
+        return firstVehiclesToOtherLocationCount;
+    }
+    public set firstVehiclesToOtherLocationCount(value: number) {
+        firstVehiclesToOtherLocationCount = value;
+    }
+
     public sendAlarmGroup(alarmGroup: AlarmGroup) {
         const targetTransferPoint = selectStateSnapshot(
             createSelectTransferPoint(this.targetTransferPointId!),
             this.store
         );
+
+        const firstAlarmGroupVehiclesToOtherLocationCount =
+            this.firstVehiclesToOtherLocationCount;
+        this.firstVehiclesToOtherLocationCount = Math.max(
+            0,
+            this.firstVehiclesToOtherLocationCount -
+                Object.keys(alarmGroup.alarmGroupVehicles).length
+        );
+
         // TODO: Refactor this into one action (uuid generation is currently not possible in the reducer)
```

Comment:
As UUID generation in the reducers is now possible and you're also changing lots of the code, have you considered doing this refactoring?

Reply:
Done in d81b5f0823d86c3041b6c8f305072b1fcd177014